src/component/App.js

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+import { BrowserRouter, Routes, Route } from 'react-router-dom';
+import Home from './Home/Home';
+import LogIn from './Home/Login';
+import Register from './Home/Signup';
+import Main from './Main/Main';
+import Reserve from './Main/Forms/Reserve';
+import MyReservation from './Main/Forms/MyReservation';
+import AddCar from './Main/Forms/AddCar';
+import RemoveCar from './Main/Forms/RemoveCar';
+import Detail from './Main/Detail';
+import ReserveDetail from './Main/Forms/ReserveDetail';
+import PrivateRoute from './Home/PrivateRoute';
+
+function App() {
```

Comment:
- [ ] OPTIONAL: The frontend homepage styling is quite different from the provided design.  I suggest you touch up your CSS a little more so the app looks more like the provided design. 👍